variables.tf

```diff
@@ -1,4 +1,77 @@
-variable "example" {
-  description = "Example variable"
-  default     = "hello world"
+#-----------------------------------------------------------------------------------------------------------------------
+# OPTIONAL PARAMETERS
+# These variables have defaults set that can be overridden for desired behavior.
+#-----------------------------------------------------------------------------------------------------------------------
+
+variable "enabled_standards" {
+  description = <<DOC
```

Comment:
In order to indent I thought it had to be <<-DOC

? 

On my phone, so excuse lack of formatting

Reply:
Hmm...you're right, that's what the doc says, but this works just fine...I'll update...

Reply:
13b987955e042b3b63744bc77745f7a03bc93cca